roxie/ccd/ccdfile.cpp

```diff
@@ -913,10 +913,23 @@ class CRoxieFileCache : implements IRoxieFileCache, implements ICopyFileProgress
             if (modified.isNull()) 
                 return FileIsValid;
             CDateTime mt;
-            if (f->getTime(NULL, &mt, NULL) &&  mt.equals(modified, false))
-                return FileIsValid;
+            if (f->getTime(NULL, &mt, NULL))
+            {
+                if (fileTimeFuzzySeconds)
+                {
+                    time_t mtt = mt.getSimple();
+                    time_t modt = modified.getSimple();
+                    long diff = mtt-modt;
```

Comment:
Slightly concerned about this code - time_t can be 64bit.  Windows compilers might warn about type restriction losing information.